app/views/foreman_ansible/job_templates/convert_to_rhel.erb

```diff
@@ -0,0 +1,45 @@
+<%#
+name: Convert to RHEL
+snippet: false
+template_inputs:
+- name: Activation Key
+  required: true
+  input_type: user
+  advanced: false
+  value_type: plain
+  resource_type: Katello::ActivationKey
+  hidden_value: false
+- name: Restart
+  required: true
+  input_type: user
+  options: "yes\r\nno"
+  advanced: false
+  value_type: plain
+  resource_type: Katello::ActivationKey
+  hidden_value: false
+model: JobTemplate
+job_category: Convert 2 RHEL
+provider_type: Ansible
+kind: job_template
+%>
+<%- activation_key = input('Activation Key').blank? ? "rhel#{@host.os.major}" : input('Activation Key') -%>
```

Comment:
I am wondering if this line is needed. Activation key is required, so it should never be blank.

Reply:
ah good point, I'll drop this and repush